src/Statement/StatementGuidParser.php

```diff
@@ -7,6 +7,11 @@
 use Wikibase\DataModel\Statement\StatementGuid;
 
 /**
+ * A parser capable of splitting a statement id into the entity id of the entity the statement
+ * belongs to, and the randomly generated global unique identifier (GUID).
+ *
+ * @see Wikibase\DataModel\Statement\StatementGuid
```

Comment:
No need for the fully qualified name as this is already imported some lines above.